reels/fluter/__init__.py

```diff
@@ -19,15 +19,17 @@
 
 
 class Fluter:
-    def __init__(self, host=None, width=16, height=24, depth=3):
+    DEFAULT_HOST = None
```

Comment:
Why not set it to the actual default host? This way IDEs would be able to resolve this string.

Reply:
I am using the `None` value to know when to fall back to the environment variable (and, if I ever implement it, a command line arg).
If I did use `DEFAULT_HOST` here, I could not decide if the caller did omit the value (which means, use default of no envvar), or if they explicitly supplied the value used as `DEFAULT_HOST` as host in constructor call (which means: use this value).
I know, it is not documented anywhere, but the behaviour is kinda intuitive for me:
use default, overridden by envvar, overridden by cmdline arg, overridden by explicit value

(maybe my intention is not clear from the code, though)

Reply:
I feel that it makes the code less transparent and also that the class `Fluter` does things it shouldn't be concerned with (i.e. the environment). Shouldn't the class just care about its inputs at initialization time?

I would much prefer a solution like https://stackoverflow.com/a/10551190/1643939 where CLI and override via env vars is handled by `argparse`. That way all the option handling is on the user side and can still be abstracted away for convenience:

```python
# maybe in fluter.py
class EnvDefault(argparse.Action):
    # [...]

def fluter_argparser():
    parser = argparse.ArgumentParser()
    parser.add_argument(
        "--host", action=EnvDefault, envvar='FLUTER_HOST', 
        help="Address (and port) of the DeepPixelFlut server")
    return parser

# in a reel
parser = fluter_argparser()
args = parser.parse_args()

fluter = Fluter(host=args.host)
```

But if you think this is not worth the hassle then we can just merge this PR.